File: Examples/CSharp/Text/ParagraphsAlignment.cs
using System.IO;

using Aspose.Slides;
using Aspose.Slides.Export;

namespace Aspose.Slides.Examples.CSharp.Text
{
    public class ParagraphsAlignment
    {
        public static void Run()
        {
            // ExStart:ParagraphsAlignment
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_Text();

            // Instantiate a Presentation object that represents a PPTX file
            using (Presentation pres = new Presentation(dataDir + "ParagraphsAlignment.pptx"))
            {

                // Accessing first slide
                ISlide slide = pres.Slides[0];

                // Accessing the first and second placeholder in the slide and typecasting it as AutoShape
                ITextFrame tf1 = ((IAutoShape)slide.Shapes[0]).TextFrame;
                ITextFrame tf2 = ((IAutoShape)slide.Shapes[1]).TextFrame;

                // Change the text in both placeholders
                tf1.Text = "Center Align by Aspose";
                tf2.Text = "Center Align by Aspose";

                // Getting the first paragraph of the placeholders
                IParagraph para1 = tf1.Paragraphs[0];
                IParagraph para2 = tf2.Paragraphs[0];

                // Aligning the text paragraph to center
                para1.ParagraphFormat.Alignment = TextAlignment.Center;
                para2.ParagraphFormat.Alignment = TextAlignment.Center;

                //Writing the presentation as a PPTX file
                pres.Save(dataDir + "Centeralign_out.pptx", SaveFormat.Pptx);
            }
            // ExEnd:ParagraphsAlignment
        }
    }
}

File: Examples/VisualBasic/Text/ParagraphsAlignment.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Slides
Imports Aspose.Slides.Export

Namespace Aspose.Slides.Examples.VisualBasic.Text
    Public Class ParagraphsAlignment
        Public Shared Sub Run()
            ' ExStart:ParagraphsAlignment

            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_Text()

            ' Instantiate a Presentation object that represents a PPTX file
            Using pres As New Presentation(dataDir & "ParagraphsAlignment.pptx")

                ' Accessing first slide
                Dim slide As ISlide = pres.Slides(0)

                ' Accessing the first and second placeholder in the slide and typecasting it as AutoShape
                Dim tf1 As ITextFrame = (CType(slide.Shapes(0), IAutoShape)).TextFrame
                Dim tf2 As ITextFrame = (CType(slide.Shapes(1), IAutoShape)).TextFrame

                ' Change the text in both placeholders
                tf1.Text = "Center Align by Aspose"
                tf2.Text = "Center Align by Aspose"

                ' Getting the first paragraph of the placeholders
                Dim para1 As IParagraph = tf1.Paragraphs(0)
                Dim para2 As IParagraph = tf2.Paragraphs(0)

                ' Aligning the text paragraph to center
                para1.ParagraphFormat.Alignment = TextAlignment.Center
                para2.ParagraphFormat.Alignment = TextAlignment.Center

                'Writing the presentation as a PPTX file
                pres.Save(dataDir & "Centeralign_out.pptx", SaveFormat.Pptx)
            End Using
            ' ExStart:ParagraphsAlignment
        End Sub
    End Class
End Namespace